TESTING-INSTRUCTIONS.md

```diff
@@ -2,6 +2,17 @@
 
 ## Unreleased
 
+### Add filter to profile wizard steps #6564
+
+1. Remove a step via the filter.
+```
+addFilter( STEPS_FILTER, 'woocommerce-admin', ( steps ) => {
```

Comment:
I'm wondering if we should include this test for this at all. I'd imagine testers would have a hard time testing this since it requires some understanding of where to put the code, setting up local env, and running a build.

@woocommerce/isotope I would love to hear if folks have any opinion on this

Reply:
That's a good point @ilyasfoo.  I updated the instructions in https://github.com/woocommerce/woocommerce-admin/pull/6564/commits/827f4b4ed2f84f7162e12ecf80774011ace48af2 to create an easier way to add this snippet on a JN site to make sure it's working correctly.  Also updated the code to use the wp hooks global so no modern build is needed.

Reply:
Ah, I haven't thought about that, makes sense!